core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -338,34 +276,14 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
       } yield ()
     }
 
-    val userDirectoryRemoval = for {
-      _ <- userDirectoryDAO.disableUser(userRef.userSubjectId) recover { case _ => () }   // may not be strictly necessary, but does not hurt
-      _ <- userDirectoryDAO.removeUser(userRef.userSubjectId) recover { case _ => () }
-    } yield ()
-
     val proxyGroupDeletion = userF.flatMap(gcsDAO.deleteProxyGroup) recover { case e: HttpResponseException if e.getStatusCode == 404 => Unit }
 
     for {
-      _ <- Future.sequence(Seq(dbTablesRemoval, userDirectoryRemoval, proxyGroupDeletion, deleteRefreshTokenInternal(userRef)))
+      _ <- Future.sequence(Seq(dbTablesRemoval, proxyGroupDeletion, deleteRefreshTokenInternal(userRef)))
       _ <- deleteUserFromDB(userRef)
     } yield RequestComplete(StatusCodes.NoContent)
   }
 
-  def addToLDAP(userSubjectId: RawlsUserSubjectId): Future[PerRequestMessage] = {
```

Comment:
Do this and `removeFromLDAP` need to be re-implemented in sam? We can just do `ldapadd`s to get people in there if we need to